.github/code_owners.yml

```diff
@@ -48,7 +48,6 @@ packages/loader/**:
 - anthony-murphy
 packages/runtime/**:
 - vladsud
```

Comment:
i wonder if this support github teams. i always hate having aliases in code

Reply:
#8701 tracks switching to groups and GitHub's native CODEOWNERS features.